### DISCUSSION TITLE
How to add a description in manifest?

### DISCUSSION THREAD

**Discussion by JenuelDev** [vuejs/vue-cli#6460] [Q&A] [answered]
I just want to add a description in my manifest to show it in the browser's application details. When opening the dev tool.
![image](https://user-images.githubusercontent.com/31676163/117437381-b1d34180-af62-11eb-80cb-aa64993cd879.png)

**Comment by kefir500** [comment, marked as answer]
```js
module.exports = {
  pwa: {
    manifestOptions: {
      description: 'Your description goes here', // Add this to your vue.config.js
    },
  },
};
```
